internal/fields/exceptionfields.go

```diff
@@ -37,6 +37,8 @@ func (v *Validator) listExceptionFieldsMapElement(root string, elem common.MapSt
 			all = append(all, fields...)
 		default:
 			if skipLeafOfObject(root, name, v.specVersion, v.Schema) {
+				logger.Warnf("Skip validating leaf of object (key %q spec %q)", key, v.specVersion)
```

Comment:
Should we remove these kind of messages? Do you think they are adding noise to the output ?

And, should these other warnings be removed too? 

https://github.com/elastic/elastic-package/blob/b64f3748e2b513f3ee8baa4474f21b209a937162/internal/fields/mappings.go#L564

https://github.com/elastic/elastic-package/blob/b64f3748e2b513f3ee8baa4474f21b209a937162/internal/fields/mappings.go#L575

https://github.com/elastic/elastic-package/blob/b64f3748e2b513f3ee8baa4474f21b209a937162/internal/fields/mappings.go#L440

https://github.com/elastic/elastic-package/blob/b64f3748e2b513f3ee8baa4474f21b209a937162/internal/fields/exceptionfields.go#L89

Reply:
Probably too much verbose, look at these output:
https://buildkite.com/elastic/integrations/builds/23034#01956614-a005-472f-a3aa-91d8f5940371/6-9

Reply:
Yes, this is probably too verbose, maybe even at the debug level.

If we want to have this at the debug level, we could print the full list after collecting it instead of once per element.

Reply:
I added a new TRACE log level here https://github.com/elastic/elastic-package/pull/2454/commits/fc6c793420c0b4e7f3ea8e7c84799f7a19f050d6 if more than `-v` is set in the command line.

This log level should not be enabled in CI.

Probably, this would also help in other scenarios.

WDYT ? @jsoriano